### ISSUE TITLE
colour palette for the design

### ISSUE THREAD

**Issue by HelChris** [NoroffFEU/ai-project#109]
**Description:**
The app should have a consistent colorpalett to be more appealing to the user. 
Use the moodboard inside figma for inspiration: 
https://www.figma.com/design/aLPwzLS6nF7BHJPlf1aQyx/Design---AI-Team?node-id=4-2&p=f&t=Qy8yrcmNAVoewr2H-0

## Acceptance Criteria
1. the colorpalett must be WCAG approved regaring contrast etc
2. the color is added to _variables.scss file to the correct variable name with the hexacode for that color 
3. the _variables.scss files colorpalette is updated with the chosen colors

<img width="1185" height="1013" alt="Image" src="https://github.com/user-attachments/assets/9531c9f8-77ff-4fbf-ac0f-471b2d7a9b08" />
